propulsion_motor/src/propulsion_motor.cpp

```diff
@@ -7,8 +7,7 @@
 #include "rover_can_lib/msgs/propulsion_motor_cmd.hpp"
 #include "rover_can_lib/msgs/propulsion_motor_status.hpp"
 
-#include "actuators/talon_srx.hpp"
-#include "actuators/motor_driver.hpp"
+#include "actuators/motor_drivers/IFX007T.hpp"
```

Comment:
Should this be included if not tested (even if error is managed in IFX007T class)?

Reply:
IFX007T class was tested and is used in the main code